src/main/drivers/adc.h

```diff
@@ -24,7 +24,7 @@ typedef enum {
     ADC_CURRENT = 1,
     ADC_EXTERNAL1 = 2,
     ADC_RSSI = 3,
-    ADC_CHANNEL_MAX = ADC_CURRENT
+    ADC_CHANNEL_MAX = ADC_RSSI
```

Comment:
nice find. can we just remove all the assignments in this enum? seems like these are just the default values per http://c0x.coding-guidelines.com/6.7.2.2.html